data-store/db-client/migrations/20231007213152_person.down.sql

```diff
@@ -0,0 +1 @@
+drop schema if exists person cascade;
```

Comment:
Is there a way to simplify the naming convention for the migrations? To have, for example, `V1_PERSON_DOWN.sql` instead of the current name with the timestamp?

Reply:
As far as i know - no. Those weren't created by hand, but as described in readme the migration plugin names it like that